README.md

```diff
@@ -124,6 +139,59 @@ Example:
 }
 ```
 
+### Fetch content by delivery key (via Content Delivery Service 2)
+
+**Note:** In order to get content by it's delivery key via `getContentItemByKey()`, you must supply the `hubName` option to the client, you [find your Hub name under the settings section in Dynamic Content](https://docs.amplience.net/development/contentdelivery/readme.html#hubname).
```

Comment:
typos: it's instead of its above and below item item, using SDK. 
Use of id over key in 'If no content is found with the provided id the promise will reject with an error'?

Reply:
fixed.